Nodejs/Product/ProjectWizard/ImportWizard.cs

```diff
@@ -45,6 +46,8 @@ public sealed class NewProjectFromExistingWizard : IWizard {
                     dte = new ServiceProvider(provider).GetService(typeof(DTE)) as DTE;
                 }
             }
+
+            bool addingNewProject = false;
```

Comment:
You're also not actually recording or using the value of AddingNew project anywhere.